src/Sandbox.php

```diff
@@ -532,7 +532,7 @@ public function setExpirationAfterHours(int $expirationAfterHours): self
         return $this;
     }
 
-    public function getExpirationAfterHours(): ?int
+    public function getExpirationAfterHours(): int
```

Comment:
> takže bych řekl že nikdy nemůže nastat situace, že expirationAfterHours je null

Muze, pokud udelas treba `(new Sandbox())->getExpirationAfterHours()`. Je to edge-case, ale technicky mozny (https://github.com/keboola/sandboxes/blob/36abe6e826be7bc8628972e5c671e16d1f03771d/src/Library/SandboxFactory.php#L22)

Reply:
Jsem zil v domeni ze kdyz je ta property bez default hodnoty
```
private int $expirationAfterHours;
```

tak to pak hodi chybu kdyz jsou zaply strict types a neni default hodnota

Reply:
Z API se to vsude prevadi pres to `Sandbox::fromArray()` a tam se vsude misto null natavi 1

Reply:
Ahaa, prehlidl jsme, ze ta property neni nullable. To neznamena, ze by tam ten edge-case neni, ale je tma uz davno :smile: a ta tvoje zmena to nijak nezhorsi :+1: